src/watts/plugin_moose.py

```diff
@@ -106,54 +106,60 @@ def _from_hdf5(cls, obj: h5py.Group):
         return cls(parameters, time, inputs, outputs)
 
 
-class PluginSAM(TemplatePlugin):
-    """Plugin for running SAM
+class PluginMOOSE(TemplatePlugin):
+    """Plugin for running MOOSE
 
     Parameters
     ----------
     template_file
-        Templated SAM input
+        Templated MOOSE input
     show_stdout
-        Whether to display output from stdout when SAM is run
+        Whether to display output from stdout when MOOSE is run
     show_stderr
-        Whether to display output from stderr when SAM is run
+        Whether to display output from stderr when MOOSE is run
 
     Attributes
     ----------
-    sam_exec
-        Path to SAM executable
+    moose_exec
+        Path to MOOSE executable
 
     """
     def  __init__(self, template_file: str, show_stdout: bool = False,
-                  show_stderr: bool = False):
+                  show_stderr: bool = False, n_cpu: int = 1, supp_inputs: List[str] = []):
```

Comment:
Can you add a description of these parameters in the docstring above?

Reply:
will do.